pkg/ocicni/ocicni.go

```diff
@@ -246,12 +339,20 @@ func (plugin *cniNetworkPlugin) TearDownPod(netnsPath string, namespace string,
 		return err
 	}
 
+	fullPodName := buildPodFullName(namespace, name)
+	plugin.podLock(fullPodName).Lock()
+	defer plugin.podUnlock(fullPodName)
+
```

Comment:
Looks like we've these three lines in a few spots.  Can we move them to their own function?

Reply:
Oops, probably not the defer.  Something to consider for the other two lines though.

Reply:
done, fixed in https://github.com/cri-o/ocicni/pull/5